src/view/draggable/draggable.jsx

```diff
@@ -188,13 +186,6 @@ export default class Draggable extends Component<Props> {
     // At this point the ref has been changed or initially populated
 
     this.ref = ref;
-
-    // After a ref change we might need to manually force focus onto the ref.
```

Comment:
focus retention logic is now controlled by the component that has focus: the drag handle